Description: Relax toml version to >=0.8
Author: Facundo Gaich <facugaich@gmail.com>
Last-Update: 2025-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: toml2json/Cargo.toml
===================================================================
--- toml2json.orig/Cargo.toml
+++ toml2json/Cargo.toml
@@ -53,7 +53,7 @@ features = ["derive"]
 version = "1.0"
 
 [dependencies.toml]
-version = "0.9"
+version = ">= 0.8, < 0.10"
 
 [profile.release]
 lto = true
